app/models/section_phrase.rb

```diff
@@ -0,0 +1,4 @@
+class SectionPhrase < ApplicationRecord
```

Comment:
Was debugging for about an hour when I realized this file name was misspelled. gahhhh.

Reply:
gotta love it!